libfirm-rs/src/nodes_gen.rs

```diff
@@ -0,0 +1,2850 @@
+use libfirm_rs_bindings as bindings;
+use std::collections::HashMap;
+
+#[derive(Debug)]
+pub enum Node {
+    ASM(ASMNode),
+    Add(AddNode),
+    Address(AddressNode),
+    Align(AlignNode),
+    Alloc(AllocNode),
+    Anchor(AnchorNode),
+    And(AndNode),
+    Bad(BadNode),
+    Bitcast(BitcastNode),
+    Block(BlockNode),
+    Builtin(BuiltinNode),
+    Call(CallNode),
+    Cmp(CmpNode),
+    Cond(CondNode),
+    Confirm(ConfirmNode),
+    Const(ConstNode),
+    Conv(ConvNode),
+    CopyB(CopyBNode),
+    Deleted(DeletedNode),
+    Div(DivNode),
+    Dummy(DummyNode),
+    End(EndNode),
+    Eor(EorNode),
+    Free(FreeNode),
+    IJmp(IJmpNode),
+    Id(IdNode),
+    Jmp(JmpNode),
+    Load(LoadNode),
+    Member(MemberNode),
+    Minus(MinusNode),
+    Mod(ModNode),
+    Mul(MulNode),
+    Mulh(MulhNode),
+    Mux(MuxNode),
+    NoMem(NoMemNode),
+    Not(NotNode),
+    Offset(OffsetNode),
+    Or(OrNode),
+    Phi(PhiNode),
+    Pin(PinNode),
+    Proj(ProjNode),
+    Raise(RaiseNode),
+    Return(ReturnNode),
+    Sel(SelNode),
+    Shl(ShlNode),
+    Shr(ShrNode),
+    Shrs(ShrsNode),
+    Size(SizeNode),
+    Start(StartNode),
+    Store(StoreNode),
+    Sub(SubNode),
+    Switch(SwitchNode),
+    Sync(SyncNode),
+    Tuple(TupleNode),
+    Unknown(UnknownNode),
+}
+
+impl Into<*const ir_node> for Node {
```

Comment:
`From` instead of `Into`: 
> Library authors should not directly implement this trait, but should prefer implementing the From trait, which offers greater flexibility and provides an equivalent Into implementation for free, thanks to a blanket implementation in the standard library.

https://doc.rust-lang.org/std/convert/trait.Into.html